### PR TITLE
refactor and made it more like command

### PR DESCRIPTION
**Pull request by kryr34**
almost done
- [x] setup
- [x] get
- [x] delete
- [x] submit
- [x] status
- [x] test_status
- [x] one more line colorful !!

### CODE REVIEW

**Comment by NOOBDY** [comment]
no need to add underscores in front of files and remove unnecessary imports

**Review by NOOBDY — CHANGES_REQUESTED** [review]
changes required

File: utils/get_test_status.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from typing import Dict
+
+import urllib3
+from bs4 import BeautifulSoup
+from urllib3.exceptions import InsecureRequestWarning
+
+urllib3.disable_warnings(category=InsecureRequestWarning)
+
+
+def _get_test_status(self, id_: int, index: str) -> Dict[str, bool]:
```
Comment (NOOBDY) [inline utils/get_test_status.py:10]: shouldn't this return a List of Tuples instead?

File: utils/get_question_statuses.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import urllib3
+from bs4 import BeautifulSoup
+from dateutil import parser
+from urllib3.exceptions import InsecureRequestWarning
+
+urllib3.disable_warnings(category=InsecureRequestWarning)
+
+
+def _get_question_statuses(self) -> Dict[str, Dict[str, str]]:
```
Comment (NOOBDY) [inline utils/get_question_statuses.py:12]: also here

File: utils/get_test_status.py (hunk 1 above)
Comment (kryr34) [reply utils/get_test_status.py:10]: for some cases index may not be sequence

**Pull request merged**